src/dftbp/dftbplus/oldcompat.F90

```diff
@@ -904,7 +904,7 @@ subroutine convert_13_14(root)
         call getChildValue(ch2, "", iOrder, child=ch3)
         if (iOrder > 1) then
           write(strtmp,"(A,I0,A,I0,A)")"Older Methfessel-Paxton requested order of ", iOrder,&
-              & " is now equvalent to ", (iOrder -1), " from parser version 14."
+              & " is now equivalent to ", (iOrder -1), " from parser version 14."
```

Comment:
Should really be a separate PR, but thanks for fixing.

Reply:
I know... :sweat_smile: